bin/varnishd/common/common_vsc.c

```diff
@@ -183,6 +183,20 @@ VRT_VSC_Alloc(struct vsmw_cluster *vc, struct vsc_seg **sg,
 	return (vsg->ptr);
 }
 
+void *
+VRT_VSC_AllocVariadic(struct vsmw_cluster *vc, struct vsc_seg **sg,
```

Comment:
I would have suggested `VRT_VSC_Allocv()` to match what we do in other places but in this case it's completely backwards :-/